Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -19,20 +19,26 @@ import class Foundation.Timer
 import struct Foundation.TimeInterval
 import protocol Foundation.StreamDelegate
 
-/// Reads data from a smithy-swift native `ReadableStream` and streams the data to a Foundation `InputStream`.
+/// Reads data from a smithy-swift native `ReadableStream` and streams the data through to a Foundation `InputStream`.
+///
+/// A pair of Foundation "bound streams" is created.  Data from the `ReadableStream` is transferred into the Foundation
+/// `OutputStream` until the `ReadableStream` is closed and all data has been read from it.  The Foundation
+/// `InputStream` is exposed as a property, and may be used to stream the data to other components.
```

Comment:
Lots of added commentary in this file to help guide the reader through how this type works.